packages/user/integration-tests/__tests__/services/module/invite.spec.ts

```diff
@@ -188,5 +214,26 @@ describe("UserModuleService - Invite", () => {
         })
       )
     })
+
+    it("should emit invite created events", async () => {
+      const eventBusSpy = jest.spyOn(MockEventBusService.prototype, "emit")
+      await service.createInvites(defaultInviteData)
+
+      expect(eventBusSpy).toHaveBeenCalledTimes(1)
+      expect(eventBusSpy).toHaveBeenCalledWith([
+        expect.objectContaining({
+          body: expect.objectContaining({
+            data: "1",
```

Comment:
suggestion: data should probably be an object which contains the id property no?